backends-common/elasticsearch/src/test/resources/auth-es/nginx-conf/passwd

```diff
@@ -0,0 +1 @@
+elasticsearch:$apr1$m5FvKpw7$EZ9UEUCilHyKTIqQ57nWG1
```

Comment:
Please have a comment, or a README next to this file giving the password in plain text...